pkg/adapter/github/adapter.go

```diff
@@ -51,133 +56,150 @@ func (ra *Adapter) HandleEvent(payload interface{}, header webhooks.Header) {
 
 func (ra *Adapter) handleEvent(payload interface{}, hdr http.Header) error {
 	if ra.client == nil {
```

Comment:
Not your code so I don't blame you @n3wscott, but: this source and several others modified here construct the client in the handle function. That's fine, but these implementations aren't thread-safe. I'll create an issue.

Reply:
#227